region/norfair/crocomire.json

```diff
@@ -1621,6 +1622,19 @@
                     "Obstacle B is present here because you need access to node 2 to bring a Mella over.",
                     "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
                   ]
+                },
+                {
+                  "name": "Indy Jones Mella Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "canMellaIceClip",
+                    "canWalljump",
+                    "canCrumbleSpinJump"
```

Comment:
There are ways around the crumble jump.  Falling on the mella and jumping again, refreezing the mella above the crumble, using xray standup.  I don't think it should be a requirement.

Reply:
Thanks, I removed this.